### ISSUE TITLE
ssh Remote connections do not support single quotes

### ISSUE THREAD

**Issue by jianboy** [gitpod-io/gitpod#20073]
### Bug description

first, ssh client did not support single quotes:

this is error, can not find host:
![image](https://github.com/user-attachments/assets/b489f7e2-14ee-4ac6-9532-fdcb557fe281)

this is ok:
![image](https://github.com/user-attachments/assets/bcad5757-5e33-44c3-9e77-6523524ed2a0)

but website show this command:
![image](https://github.com/user-attachments/assets/795cb21e-07bf-4bd6-a3b7-974f6f5d8c4f)

second, I can not connect remote gitpod container with vscode. the error is:
![image](https://github.com/user-attachments/assets/c8f70722-c54e-4089-b49b-af65442b062d)

error log:

![image](https://github.com/user-attachments/assets/5b28a4cc-ece0-4946-859a-a082b6fbb513)

```
2024-07-31 18:18:19.082 [error] Local SSH: failed to connect to beige-impala-hjioru2d Gitpod workspace: Failed to call public API: [internal] Protocol error
2024-07-31 18:18:19.211 [error] Failed to connect to beige-impala-hjioru2d Gitpod workspace: Failed to call public API: [internal] Protocol error

```

### Steps to reproduce

just open a gitpod project.

**Comment by jianboy** [comment]
I think the error is caused by ssh not supporting single quotes, so replace single quotes to double quotes in gitpod plugins.

**Comment by axonasif** [comment]
What's the output of `ssh -V`?

**Comment by jianboy** [comment]
> What's the output of `ssh -V`?

`OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2`

**Comment by axonasif** [comment]
I'm not sure why this is happening, can you try updating it? Was OpenSSH installed by default?

**Comment by jianboy** [comment]
I didn't know how to update it(OpenSSH). this is a little problem.

the real problem is that I can not refresh repo list in vscode  with gitpod plugin.

https://github.com/user-attachments/assets/a2fffd94-0b74-49c0-940b-d3ac181155c9

**Comment by jianboy** [comment]
I think the problem is that the gitpod extension does not support single quotes.

**Comment by axonasif** [comment]
> the real problem is that I can not refresh repo list in vscode with gitpod plugin.

Can you follow this?

https://www.gitpod.io/docs/help/troubleshooting#gitpod-logs-in-vs-code-browser-and-desktop

**Comment by jianboy** [comment]
I found the problem. It was caused by my other plugins that could not display the list. I disabled all the plugins. the gitpod works well.

These are actually two problems. The lack of support for single quotes does exist. The plug-in cannot be used because of other plug-ins, which I have uninstalled.

[vscode-desktop-logs-20240806T145244.zip](https://github.com/user-attachments/files/16512197/vscode-desktop-logs-20240806T145244.zip)
